notebooks/basics/multiple-systems.ipynb

```diff
@@ -804,59 +842,64 @@
+    "\n",
+    "Suppose $(\\mathsf{X}, \\mathsf{Y})$ is in a tensor product state $\\vert \\phi\\rangle \\vert\\psi\\rangle$. If $M$ acts on $\\mathsf{X}$ and $N$ acts on $\\mathsf{Y}$, we should then expect the joint state to change as $M \\vert \\phi \\rangle N\\vert\\psi\\rangle$. Therefore, we define the tensor product operation $M \\otimes N$ to act on tensor product states in precisely this way.\n",
+    "\n",
+    "$$\n",
+    "  (M \\otimes N)\n",
+    "  \\vert \\phi \\rangle \\otimes \\vert \\psi \\rangle \n",
+    "  = M \\vert\\phi\\rangle \\otimes \n",
+    "  N \\vert\\psi\\rangle\n",
+    "$$\n",
     "\n",
-    "Now suppose that we have multiple systems and we *independently* perform *separate operations* on the systems.\n",
+    "This specification uniquely defines $M \\otimes N$ by linearity. Any state that is not a tensor product state can nevertheless be written as a linear combination of standard basis vectors, which are themselves tensor product states. The action of a linear operator on a basis uniquely defines that operator. \n",
     "\n",
-    "For example, taking our usual set-up of two systems $\\mathsf{X}$ and $\\mathsf{Y}$ having classical state sets $\\Sigma$ and $\\Gamma$, respectively, let us suppose that we perform one operation on $\\mathsf{X}$ and, completely independently, another operation on $\\mathsf{Y}$.\n",
-    "As we know from the previous lesson, these operations are represented by stochastic matrices — and to be precise, let us say that the operation on $\\mathsf{X}$ is represented by the matrix $M$ and the operation on $\\mathsf{Y}$ is represented by the matrix $N$.\n",
-    "Thus, the rows and columns of $M$ have indices that are placed in correspondence with the elements of $\\Sigma$ and, likewise, the rows and columns of $N$ correspond to the elements of $\\Gamma$.\n",
+    "Just as for vectors, the tensor product just defined possesses several nice properties, particularly bilinearity.\n",
     "\n",
-    "A natural question to ask is this: if we view $\\mathsf{X}$ and $\\mathsf{Y}$ together as a single, compound system $(\\mathsf{X},\\mathsf{Y})$, what is the matrix that represents the combined action of the two operations on this compound system?\n",
-    "To answer this question, we must first introduce the tensor product of matrices — which is similar to the tensor product of vectors and is defined analogously.\n",
+    "\\begin{align}\n",
+    "    M \\otimes (\\alpha N + \\beta O) &= \\alpha (M \\otimes N) + \\beta (M \\otimes O) \\\\\n",
+    "    (\\alpha M + \\beta N) \\otimes O &= \\alpha (M \\otimes O) + \\beta (N\\otimes O)\n",
+    "\\end{align}\n",
```

Comment:
Let's not use $O$ for a matrix... $N_1$ and $N_2$ or $M_1$ and $M_2$ would be better.